src/xcode/ENA/ENA/Source/Scenes/Home/Cells/Statistics/HomeStatisticsTableViewCell.swift

```diff
@@ -9,6 +9,17 @@ class HomeStatisticsTableViewCell: UITableViewCell {
 
 	// MARK: - Overrides
 
+	override var accessibilityElements: [Any]? {
+		get {
+			stackView.arrangedSubviews
+		}
+
+		// swiftlint:disable:next unused_setter_value
+		set {
+			preconditionFailure("This property is computed by the contents of the stack view")
```

Comment:
Maybe just LOG the error? wont this crash the app ?

Reply:
This is a developer error because of the overall idea to pile all elements onto one big heap. The user should never see this because DEVs should handle this properly, i.e. not `set` that value.

Reply:
No i mean just logging it not showing an alert this way we can debug later but don't block the functionality of the app

Reply:
Devs are lazy. If it's not exploding into their face, they won't touch it.